tools/fanatec_led_server.py

```diff
@@ -4,31 +4,45 @@
 import time
 import threading
 
+sys.path.append("../dbus")
+import fanatec_input
+
 LEDS = 9
 
+
 def set_leds(values):
     global wheel
     wheel.RPM = values
 
+
 def set_display(value):
     global wheel
     wheel.Display = value
 
+
 def set_pedals_rumble(abs, tc):
     global pedals
-    pedals.rumble = '%i'%( (0xff if tc else 0) << 16 | (0xff if abs else 0) << 8 )
+    pedals.rumble = "%i" % ((0xFF if tc else 0) << 16 | (0xFF if abs else 0) << 8)
+
 
 # clear leds and display
 def clear():
-    set_leds([False]*9)
+    set_leds([False] * 9)
     set_display(-1)
 
 
 class Client(threading.Thread):
-    def __init__(self, ev, dbus=True, device=None, display='gear'): 
+    def __init__(
+        self,
+        ev,
+        dbus=True,
+        device=None,
+        display="gear",
+        wheel=fanatec_input.CSLEliteWheel,
```

Comment:
Maybe don't specify the wheel by argument, but look it up by using the `wheel_id` sysfs file

Reply:
added in the next version!